src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -82,6 +85,51 @@ struct SimplexResult
     mpsWriterFactory mps_writer_factory;
 };
 
+static void fillModelerComponents(std::vector<std::unique_ptr<ComponentFiller>>& componentFillers,
+                                  std::vector<LinearProblemFiller*>& fillersCollection,
+                                  const Antares::Study::SystemModel::System* modelerSystem)
+{
+    if (modelerSystem == nullptr)
+    {
+        return;
+    }
```

Comment:
Do we need this **if** block ?
From what I can see, it's constructed in `System SystemBuilder::build()  const`, and this seems to guarantee a non null reference.

Reply:
If there's no modeler files the object is not initialized -> segfault